src/main/java/com/feed_the_beast/ftbu/gui/GuiClaimedChunks.java

```diff
@@ -67,13 +68,16 @@ public void onClicked(IGui gui, IMouseButton button)
             boolean claim = !GuiScreen.isShiftKeyDown();
             boolean flag = button.isLeft();
 
-            if(flag)
+            if(FTBUConfigWorld.CHUNK_CLAIMING.getBoolean())
```

Comment:
There should be no changes to this file, client side doesn't know what configs are on server side. And server decides what to do with claimed chunks anyway